edits to chla files

estumpnerUSGS opened the pull request:
@goertler @catarfish I added Pascale's code back into f_load_chla (remove duplicates, assign regions and choose 1 measurement at random. I noticed EMP EZ2 is not identified in stations in the 'main below' region (n=96)

goertler commented:
Hey Liz - did Ryan help revert the f_load_chla to add my code back? Sorry to be a pain, but because this is our final product we should be following the reproducible best practices.

estumpnerUSGS commented:
HI Pascale,

I didn't have Ryan revert anything. Below is a summary of the sequence of events and my argument against reverting the code.
This is all a long way of saying that I don't think this compromises the spirit of reproducibility. It's true our collaboration for the f_load_chla function is not fully captured on Github, but we did collaborate and what we did land on - that is reproducible - is the final code.  Several of the steps leading up to the final product were mistakes on my part.

I edited the f_load_chla prior to the YBFMP data publication to EDI. If you recall, I added the script to the repo via a direct upload (not through a reviewed pull request) bc I was having trouble installing Git in RStudio on my new machine.

You added your code to f_load_chla that deleted duplicates and created a hierarchy of stations. You created a pull request that I did not merge.

Then, the YBFMP data was published and the f_clean_YBFMP and f_load_chla had to be edited again.

I edited the scripts and replaced them with another manual upload (I think) but the f_load_chla did not have your code in it. I also resolved Git in this time period but didn't know how to stash old items in Git before committing changes to the scripts. That is also why I completed the manual upload a second time.

You, Cat, and I met and I brought up your code did not exist in the scripts. We talked about reverting pull requests. You also pointed me to how resolve Git stash stuff.

I resolved Git stash issues. I removed any manual uploads from the data_published folder. I copied your code in the unmerged pull request and added it to f_load_chla. I ran it to produce the final chla file that Cat could in turn use.


Talk to you soon.

Liz

From: Pascale ***@***.***>
Sent: Friday, May 12, 2023 10:34 AM
To: Delta-Stewardship-Council/swg-21-connectivity ***@***.***>
Cc: Stumpner, ***@***.*** ***@***.***>; Author ***@***.***>
Subject: Re: [Delta-Stewardship-Council/swg-21-connectivity] edits to chla files (PR #153)


Hey Liz - did Ryan help revert the f_load_chla to add my code back? Sorry to be a pain, but because this is our final product we should be following the reproducible best practices.

-
Reply to this email directly, view it on GitHub<https://github.com/Delta-Stewardship-Council/swg-21-connectivity/pull/153#issuecomment-1546068844>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AVETRWYJVMN227VDM3XEI2TXFZYAPANCNFSM6AAAAAAXSHADPA>.
You are receiving this because you authored the thread.Message ID: ***@***.******@***.***>>
